package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@easypost/api",
-  "version": "5.3.0",
+  "version": "5.5.0",
```

Comment:
🤷‍♂️  not a necessary change. The lockfile basically is a snapshot of what/when dependencies were installed/updated. The version of the lib here doesn't affect anything and serves as a moment in time to know when last all the things got bumped (2 versions ago). It's the version in the `package.json` file that is important but isn't going to get bumped just yet.

Reply:
Is this not part of our .gitignore, and should it be?

Reply:
No, you definitely want lock files checked into the repo. This is what ensures that what you and I install on separate machines is the exact same. Because in the `package.json` file we do loose version pinning so that the next time we run `npm update` it updates the dependencies based on how loose our pinning is and then updates the lockfile to reflect that. But in-between dependency bumps, it will keep a consistent set of dependencies for everyone, everytime they do `npm install` - which is ideal. Python and other languages have similar constructs but we don't use them (with the exception of Go which is the `go.sum` file and in PHP with the `composer.lock` file).

Reply:
So, do I need to roll this file back or not?

Reply:
I take your approval as a no.

Reply:
The change doesn't hurt, doesn't help - would suggest not making it in the future as it doesn't provide value but won't block on it 🙂